token/cli/src/command.rs

```diff
@@ -3484,6 +3485,45 @@ pub async fn process_command<'a>(
                 )
             });
 
+            let transfer_fee_basis_point = arg_matches.values_of("transfer_fee_basis_points").map(|mut v| {
+                v.next()
+                    .unwrap()
+                    .parse::<u16>()
+                    .unwrap_or_else(print_error_and_exit)
+            });
+
+            let transfer_fee_maximum_fee = arg_matches.values_of("transfer_fee_maximum_fee").map(|mut v| {
+                v.next()
+                    .unwrap()
+                    .parse::<f64>()
+                    .unwrap_or_else(print_error_and_exit)
+            });
```

Comment:
Since we want to map this to a non-ui amount, it's a bit trickier
```suggestion
            let transfer_fee_maximum_fee = value_of::<f64>(
                arg_matches, "transfer_fee_maximum_fee").map(|v| spl_token::ui_amount_to_amount(v, decimals));
```